changelogs/fragments/service_catalog_info.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - service_catalog_info - add service catalog info plugin (https://github.com/ansible-collections/servicenow.itsm/pull/342)
```

Comment:
I think we can drop this entry since antsibull-changelog detects new plugins and modules and creates an entry for them like https://github.com/ansible-collections/servicenow.itsm/blob/c201da47a6979b5e8666024b45824a505b1abf2e/changelogs/changelog.yaml#L230